src/content/tutorial/react-step-1/react-step-1.mdx

```diff
@@ -131,7 +141,9 @@ In the `src` directory, rename `index.css` as `index.scss`. Then in `index.js` u
 
 ### Import carbon-component styles
 
-In `index.scss` import Carbon styles by adding:
+In `index.scss`, import the Carbon styles by adding the following at the top of the file:
+
+##### src/index.scss
 
 ```scss
 @import 'carbon-components/scss/globals/scss/styles.scss';
```

Comment:
Should this be using `@import '~carbon-components'` or does it not matter?

Reply:
I haven't noticed it necessary. Maybe `~` was needed with an older version of `sass-loader`?